forward_engineering/helpers/viewHelper.js

```diff
@@ -102,60 +137,34 @@ module.exports = {
 		const orReplace = schema.viewOrReplace;
 		const ifNotExists = view.viewIfNotExist;
 		const name = bucketName ? `${bucketName}.${viewName}` : `${viewName}`;
-		const createStatement = `CREATE ${orReplace && !ifNotExists ? 'OR REPLACE ' : ''}${isGlobal ? 'GLOBAL ' : ''}${isTemporary ? 'TEMPORARY ' : ''}VIEW${ifNotExists ? ' IF NOT EXISTS' : ''} ${name}`;
-		const comment = schema.description;
-		let tablePropertyStatements = '';
 		const tableProperties =
 			schema.tableProperties && Array.isArray(schema.tableProperties)
 				? filterRedundantProperties(schema.tableProperties, ['transient_lastDdlTime'])
 				: [];
 		const viewUnityTagsStatements =
 			schema.unityViewTags && getViewTagsStatement({ viewSchema: schema, viewName: name });
 
-		if (tableProperties.length) {
-			tablePropertyStatements = ` TBLPROPERTIES (${getTablePropertiesClause(_)(tableProperties)})`;
-		}
-		script.push(createStatement);
-		if (schema.selectStatement) {
-			const columnList = view.columnList ? ` (${view.columnList})` : ' ';
-			return (
-				createStatement +
-				`${columnList} ${comment ? " COMMENT '" + encodeStringLiteral(comment) + "'" : ''} ${tablePropertyStatements} AS ${schema.selectStatement};\n\n${viewUnityTagsStatements}`
-			);
-		}
-
-		if (_.isEmpty(columns)) {
-			return;
-		}
-
-		if (comment) {
-			script.push(`COMMENT '${encodeStringLiteral(comment)}'`);
-		}
-
-		if (tablePropertyStatements) {
-			script.push(tablePropertyStatements);
-		}
-
-		if (!_.isEmpty(columns)) {
-			const fromStatement = getFromStatement(_)(collectionRefsDefinitionsMap, columns);
-			const columnsNames = getColumnNames(_)(collectionRefsDefinitionsMap, columns);
-
-			if (fromStatement && columnsNames?.length) {
-				script.push(`AS SELECT ${joinColumnNames(columnsNames)}`);
-				script.push(fromStatement);
-			} else {
-				return;
-			}
-		}
-
-		if (viewUnityTagsStatements) {
-			script.push(';\n');
-			script.push(viewUnityTagsStatements);
-
-			return script.join('\n  ');
-		}
-
-		return script.join('\n  ') + ';\n\n\n\n\n';
+		return assignTemplates(ddlTemplates.createView, {
+			orReplace: orReplace && !ifNotExists ? ' OR REPLACE ' : '',
+			global: isGlobal ? 'GLOBAL ' : '',
+			temporary: isTemporary ? 'TEMPORARY ' : '',
```

Comment:
Am I correct to assume that if there will be no `orReplace` applied, the `global` will be concatenated in the [resulted statement](https://github.com/hackolade/DeltaLake/pull/241/files#diff-a16a01de85631acbade2a2eff860717be7c1af39ce8cb5f9cac73d25b4928fbdR3), for example,  like the following?
`CREATEGLOBAL ...`